apps/core/contexts/searchContext/SearchContextTypes.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import { type TripType } from '@kiwicom/margarita-config';
+
+export type Passengers = {|
+  adults: number,
```

Comment:
I think it should be OK with `+`, what do you think?

Reply:
🚀